nomenclature/cli.py

```diff
@@ -154,3 +154,42 @@ def check_region_aggregation(
         results_df.to_excel(processed_data)
     if differences:
         differences_df.reset_index().to_excel(differences, index=False)
+
+
+@cli.command("export-project")
+@click.argument("path", type=click.Path(exists=True, path_type=Path))
+@click.argument("target", type=click.Path(path_type=Path))
+@click.option(
```

Comment:
I would simplify and remove this option. If the definitions folder is not called definitions, I'm pretty sure other parts of the program would break. Unless there's a concrete use case for having a definitions folder not called _definitions_, I'd remove it.